packages/gafl-webapp-service/src/locales/en.json

```
@@ -607,11 +607,13 @@
   "server_error_if_seeing_page": "If you keep seeing this page:  ",
   "server_error_logged": "We have logged the error and we will look into it, and make sure it doesn’t happen again. However if you keep seeing this page, please send your ",
   "server_error_occured": "Sorry, an error occurred while processing your request",
-  "server_error_post_office_check_link": "Post Office branch finder service (opens in a new tab).",
-  "server_error_post_office_check": "To check the opening times of your local Post Office branch visit the ",
-  "server_error_post_office": "If you keep seeing this page, remember that you can purchase a fishing licence at your local Post Office branch.",
+  "server_error_environment_agency": "If you keep seeing this page, you can buy a fishing licence by by calling the Environment Agency:",
```

```suggestion
  "server_error_environment_agency": "If you keep seeing this page, you can buy a fishing licence by calling the Environment Agency:",
```

Looks like this typo was on Jira but we should fix it here.